zuul.d/projects.yaml

```diff
@@ -205,6 +205,18 @@
       - publish-to-galaxy
       - publish-to-automation-hub
 
+- project:
+    name: ansible-collections/ansible.netcommon
+    default-branch: main
+    templates:
+      - system-required
+      - ansible-collections-ansible-netcommon
+      - ansible-collections-juniper-junos-netconf
+      - ansible-python-jobs
+      - integrated-queue
+      - publish-to-galaxy
+      - publish-to-automation-hub
+
```

Comment:
I suggest having only system-required template which is a no-op, as you will remove the rest of the jobs from ansible-zuul-jobs for netcommon.

Reply:
+1